File: src/main/java/com/gabrielczar/semantic/dto/InterestPointDTO.java
package com.gabrielczar.semantic.dto;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.vividsolutions.jts.geom.Geometry;
import lombok.Data;

import java.math.BigInteger;

@Data
public class InterestPointDTO {
    private BigInteger id;
    private String amenity;
    private String name;
    private Double lat;
    private Double lon;

    @JsonIgnore
    private Geometry geometry;
}

File: src/main/java/com/gabrielczar/semantic/dto/InterestPointDTO.kt
package com.gabrielczar.semantic.dto;

import com.fasterxml.jackson.annotation.JsonIgnore
import com.vividsolutions.jts.geom.Geometry
import java.math.BigInteger

data class InterestPointDTO (
    val id: BigInteger,
    val amenity : String,
    val name : String,
    val lat : Double,
    val lon : Double,

    @JsonIgnore
    val geometry : Geometry
)
